tiledb/sm/rest/rest_client.h

```diff
@@ -190,6 +190,12 @@ class RestClient {
   /** Collection of extra headers that are attached to REST requests. */
   std::unordered_map<std::string, std::string> extra_headers_;
 
+  /** Array URI to redirected server mapping */
```

Comment:
Add a period at the end, like all other block comments for member variables in this file.